huaweicloud/resource_huaweicloud_cce_cluster_v3.go

```diff
@@ -238,6 +258,25 @@ func resourceClusterExtendParamV3(d *schema.ResourceData, config *Config) map[st
 	return m
 }
 
+func resourceClusterMastersV3(d *schema.ResourceData) ([]clusters.MasterSpec, error) {
+	flavorId := d.Get("flavor_id").(string)
+	mastersRaw := d.Get("masters").([]interface{})
+	if strings.Split(flavorId, ".")[1] == "s1" && len(mastersRaw) != 1 {
```

Comment:
seems the flavor likes `cce.dec.s1.small` does not applicable.

Reply:
fixed